### DISCUSSION TITLE
I am getting following error : Unable to preventDefault inside passive event listener invocation. while using react-toastify

### DISCUSSION THREAD

**Discussion by MaxVhanamane** [fkhadra/react-toastify#892] [Q&A]
I get this error when I close the toast by clicking on X button or when I click on the toast message. If it closes automatically (autoClose) I don't see any errors in console.

**Comment by gitrlk** [comment]
Have the same issue and I can't find any ressource about it. I PreventDefault() on nothing in my code.

**Comment by Errorised** [comment]
I got the error aswell. It was caused while using the browser in mobile view. It is caused by the dragable function.

**Reply by gitrlk** [reply]
I have the issue regardless of if i'm in mobile view or not.

**Comment by thangcx-3383** [comment]
I face the same issue, some how package have conflict between closeOnClick and draggale
Made it work by set `draggable = false`

**Reply by lumie31** [reply]
This suggestion works. Thanks

**Reply by codewlad** [reply]
It worked here too, thanks!

**Reply by LakshanRukantha** [reply]
This solution works. Thanks.
